### PR TITLE
Include tvs and ec in economic security risk rating 

### PR DESCRIPTION
**Pull request by sdlyy**
Resolves L2B-6161

### CODE REVIEW

File: packages/frontend2/src/server/features/data-availability/get-da-risk-entries.ts [hunk 1]
```diff
@@ -1,14 +1,30 @@
...
   return daLayers.flatMap((daLayer) =>
     daLayer.bridges.map((bridge) => ({
       slug: daLayer.display.slug,
       daLayer: daLayer.display.name,
       daBridge: toDaBridge(bridge),
-      risks: getDaRisks(daLayer, bridge),
+      risks: getDaRisks(
+        daLayer,
+        bridge,
+        getSumFor(bridge.usedIn),
```
Comment (torztomasz) [inline packages/frontend2/src/server/features/data-availability/get-da-risk-entries.ts:25]: I would extract this to a variable

File: packages/frontend2/src/server/features/data-availability/get-da-risk-entries.ts [hunk 2]
```diff
@@ -1,14 +1,30 @@
...
+        economicSecurity[daLayer.id],
```
Comment (torztomasz) [inline packages/frontend2/src/server/features/data-availability/get-da-risk-entries.ts:26]: Same for this one

**Review by torztomasz — DISMISSED** [review]

File: packages/frontend2/src/server/features/data-availability/get-da-risk-entries.ts (hunk 2 above)
Comment (sdlyy) [reply packages/frontend2/src/server/features/data-availability/get-da-risk-entries.ts]: https://github.com/l2beat/l2beat/pull/4504/commits/5f6f821ee6079b9b29cb05aafee6e7a71c8394ac

File: packages/frontend2/src/server/features/data-availability/get-da-risk-entries.ts (hunk 1 above)
Comment (sdlyy) [reply packages/frontend2/src/server/features/data-availability/get-da-risk-entries.ts]: https://github.com/l2beat/l2beat/pull/4504/commits/5f6f821ee6079b9b29cb05aafee6e7a71c8394ac

**Pull request merged**